### ISSUE TITLE
Error in `docker` plugin: invalid integer. 

### ISSUE THREAD

**Issue by anarcher** [influxdb/telegraf#58]
```
Error in plugins: unable to parse 'docker_memory_limit,command=/opt/telegraf/telegraf\ -config\ /opt/telegraf/telegraf.toml,host=ebfbff1ca0aa,id=ebfbff1ca0aa8a6b047bdc038ed0d14cfba8495f9f4eca1ce0d588c2cb1bd051,name=/influxdb09telegraf01_telegraf_run_4 value=18446744073709551615': invalid integer
```

I am not sure that value is right or not. It is too long for int. But  `telegraf` seems like that can't send metrics to `influxdb`. Maybe cause of this error. 

I ran and tested `telegraf` on `docker` container.  Is it not good for gathering metrics?

```
FROM ubuntu:14.04
RUN apt-get -y install wget
RUN wget http://get.influxdb.org/telegraf/telegraf_0.1.4_amd64.deb 
RUN dpkg -i telegraf_0.1.4_amd64.deb
ADD ./telegraf.toml /opt/telegraf/
WORKDIR /opt/telegraf
CMD ["/opt/telegraf/telegraf","-config","/opt/telegraf/telegraf.toml"]

telegraf:
    build: .
    dockerfile: telegraf.Dockerfile
    volumes: 
        - /sys:/sys:ro
        - /var/run/docker.sock:/var/run/docker.sock
```

**Comment by sparrc** [comment]
Adding a bug label, will investigate

**Comment by sparrc** [comment]
I'm marking this as closed by PR #180, please re-open if that did not fix it
